REQUIRE

```diff
@@ -2,4 +2,3 @@ julia 0.3
 Mustache
```

Comment:
Good call, will add.

Thanks (how do you keep up with changes to random _packages_ before changes even hit metadata -- you are a machine!)